src/config/seed.rs

```diff
@@ -0,0 +1,209 @@
+use crate::seed;
+use pem::{encode, Pem};
+use seed::SEED_LENGTH;
+use std::{
+    ffi::OsStr,
+    fmt,
+    fs::{self, File},
+    io::{self, Write},
+    path::{Path, PathBuf},
+};
+
+#[derive(Clone, Copy, PartialEq)]
+pub struct Seed(seed::Seed);
```

Comment:
:+1: Nice! At first I was confused with the 2 seeds, but wrapping the original one in a `config::Seed` for deserialization and serialization makes a lot of sense.

Reply:
Yes, in `comit-rs` it's called `RootSeed` but the context is different. I think it make sense to keep the "real" `Seed` agnostic on how it's configured/written on disk/etc.